workalendar/usa/core.py

```diff
@@ -14,6 +14,7 @@ class UnitedStates(WesternCalendar, ChristianMixin):
         (7, 4, 'Independence Day'),
     )
     # Veterans day label
+    include_veterans_day = True
```

Comment:
I can't see the point of this flag.
Veterans day is unanimously observed by all 50 States, there's no case where you'd have a "False" value here.

Reply:
https://github.com/Natim/Diggory/blob/master/diggory/calendar/usa.py#L9